test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -471,7 +471,23 @@ json_spirit::mObject fillBCTest(json_spirit::mObject const& _input, bool _allowI
     }
 
     output["blocks"] = blArray;
-    output["postState"] = fillJsonWithState(testChain.topBlock().state());
+
+    size_t totalCodeSize = 0;
+    bool isSmallState = testChain.topBlock().state().addresses().size() < 20;
```

Comment:
Cleaner way than setting the flag to true, then possibly setting it to false would be to move it all to separate function `isSmallState(testChain.topBlock().state())`
(with early return like
```
if (testChain.topBlock().state().addresses().size() >= 20)
  return false;

for (...)
{
  ...
  if (totalCodeSize > 1024 * 1000)
    return false;
}
return true;
```